framework-plugins/lisk-framework-http-api-plugin/src/controllers/index.ts

```diff
@@ -12,4 +12,7 @@
  * Removal or modification of this copyright notice is prohibited.
  */
 
+import * as accounts from './account';
```

Comment:
i think `export * as accounts from './account'` should work

Reply:
Done.